File: Corrsion/Cui.py
from __future__ import annotations


def main() -> None:
    C_solid = 143.0
    C_sat = 5.1

    CSe = C_solid / C_solid
    CLe = C_sat / C_solid

    print(f"CSe = {CSe:.4f}")
    print(f"CLe = {CLe:.4f}")


if __name__ == "__main__":
    main()

File: Corrsion/Cui.m
#!/usr/bin/env -S octave --no-gui --quiet
set(0, "defaultfigurevisible", "off");  % hide figure window
addpath(genpath("./../src"));  % add src folder to path




C_solid = 143; % mol/L
C_sat = 5.1; % mol/L


CSe = C_solid/C_solid;
CLe = C_sat/C_solid;

myprint("CSe", CSe);
myprint("CLe", CLe);
